docs/tutorials/google/identifying_drift.ipynb

```diff
@@ -0,0 +1,653 @@
+            "text/plain": [
+              "<Figure size 576x576 with 2 Axes>"
+            ]
+          },
+          "metadata": {
+            "needs_background": "light"
+          },
+          "output_type": "display_data"
+        }
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "cgueunn8xUXL"
+      },
+      "source": [
+        "# Simultaneous Readout\n",
+        "\n",
+        "We'll collect the Parallel P11 error data for each qubit with the internal Simultaneous Readout protocol, accessible easily with [`estimate_parallel_single_qubit_readout_errors`](https://quantumai.google/reference/python/cirq/experiments/estimate_single_qubit_readout_errors). This function runs the experiment to estimate P00 and P11 errors for each qubit (as opposed to querying for the most recent calibration data). The experiment is simple, measuring each qubit in the $|0⟩$ and $|1⟩$ states, and evaluating how often the qubit is measured in the expected state. "
+      ]
+    },
+    {
+      "cell_type": "code",
+      "metadata": {
+        "colab": {
+          "base_uri": "https://localhost:8080/",
+          "height": 532
+        },
+        "id": "o4SMWcQcjxvg",
+        "outputId": "81b2dc67-7347-471c-c743-de870803da61"
+      },
+      "source": [
+        "sq_result = cirq.estimate_parallel_single_qubit_readout_errors(sampl
```

Comment:
This analysis is quite confusing and hand-wavy.  Is this really our best practice?  Our advice is to stare at these heatmaps and try to manually glean some information out of them?  We are not going to tell them to refit bad two-qubit gates or to use any readout mitigation techniques?